kotlin/lib/src/main/kotlin/Message.kt

```diff
@@ -0,0 +1,35 @@
+package com.svix.kotlin
+
+import com.svix.kotlin.exceptions.ApiException
+import com.svix.kotlin.internal.apis.MessageApi
+import com.svix.kotlin.models.ListResponseMessageOut
+import com.svix.kotlin.models.MessageIn
+import com.svix.kotlin.models.MessageOut
+
+class Message(debugUrl: String = SvixOptions.DEFAULT_URL) {
```

Comment:
Can we make the constructor private so that others can't instantiate it?  (same for all of the rest of the models)

Reply:
Done